src/ralph/export_to_ng/resources.py

```diff
@@ -309,7 +309,7 @@ def dehydrate_deleted(self, asset):
             deleted = asset.deleted or asset.device_info.ralph_device.deleted
         except AttributeError:
             pass
-        return deleted
+        return '1' if deleted else '0'
```

Comment:
by default ``Resource`` returns 1/0 for boolean type but if you using dehydrate then you must care of this - BackOfficeAsset, Support and Licence return correct values